CHANGELOG-developer.next.asciidoc

```diff
@@ -69,6 +69,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 
 ==== Bugfixes
 
+- Fix bug in Prometheus histogram.{pull}36537[36537]
```

Comment:
Since this is user facing message we should be more specific in what the fix is about. Sth like the following (feel free to improve more):
```suggestion
- Fix how Prometheus histograms are calculated when percentiles are provide.{pull}36537[36537]
```